vs/package.json

```diff
@@ -47,32 +61,86 @@
 					"id": "activeImpactView",
 					"visibility": "collapsed",
 					"name": "Affected Objects",
-					"contextualTitle": "Active Editor",
-					"when": "vscode-sourceorbit:projectsLoaded == true"
+					"contextualTitle": "Active Editor"
```

Comment:
Why don't you want to delay until there is a SourceOrbit project loaded?

Reply:
Rather than hiding the view, I think it made more sense to always have the view visible and use welcome text.

![image](https://github.com/user-attachments/assets/7233de82-f1f9-41c9-b10a-aa8e7a3f162a)

Reply:
Should we somehow indicated that these are "Affected IBM i Objects"
This view will popup when the user is doing unrelated things now, they may open a Java files and not see "affected objects"

Reply:
Good point. Renaming the view would make sense or updating the welcome text to also say "...affected IBM i objects"

@worksofliam Any thoughts on the visibility of this view?

Reply:
@worksofliam Updated the text: [1294f4b](https://github.com/IBM/sourceorbit/pull/84/commits/1294f4b1c5181f8d040fbca1c602bae2870ceca4)

As for moving the view outside where it typically belongs, I do see that a `contextualTitle` was already added to both views which is rendered instead of the panel name:
- `Active Editor: Source Impacts`
- `Source Changes: Change Impacts`

Keep as is? Change to `IBM i: ...`? Change to `Source Orbit: ...`?

Reply:
If no change is needed, looks good to merge 😃